perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
Yes, the change I'm working on will allow partial Juju solutions -- i.e., someone could deploy the Juju-based Cassandra but run a totally different client, like cassandra_ycsb. That's the test case I'll use here.

There's potentially some issue mixing Juju and non-Juju-deployed bits, but I suspect it'll be on a case-by-case basis and we can address those if they come up, as more packages get Juju support.